cli/command/image/build_test.go

```diff
@@ -161,12 +161,16 @@ COPY data /data
 // TestRunBuildFromLocalGitHubDirNonExistingRepo tests that build contexts
 // starting with `github.com/` are special-cased, and the build command attempts
 // to clone the remote repo.
+// TODO: test "context selection" logic directly when runBuild is refactored
+// to support testing (ex: docker/cli#294)
```

Comment:
Needs a rebase now?

Reply:
Rebase will only change the diff shown in github. It will merge clean because the commit is the same.